lib/request.js

```diff
@@ -83,6 +83,8 @@ Request.prototype.prepare = function (next) {
     this._lightMyRequest.payload = payload
     return next()
   })
+
+  this._lightMyRequest.payload.resume()
```

Comment:
Can you add a comment before this line? This is happening because the payload is a stream1

Reply:
`form-data` inherits indirectly from `require('stream').Stream`

https://github.com/form-data/form-data/blob/master/lib/form_data.js#L16

https://github.com/felixge/node-combined-stream/blob/master/lib/combined_stream.js#L18

is `require('stream').Stream` the stream1?

Reply:
yes it is.